<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiveOrdersMainWidget</class>
 <widget class="QWidget" name="ReceiveOrdersMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="topWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="minimumSize">
         <size>
          <width>215</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="allRecieveButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	font-family:&quot;微软雅黑&quot;;
	font-size:14px;
	color:#333333;
	border-image: url(:/receivingGoodsImage/topTab_Press.png);
}</string>
           </property>
           <property name="text">
            <string>全部</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="noFinishReciveButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	font-family:&quot;微软雅黑&quot;;
	font-size:14px;
	color:#ffffff;
	border-image: url(:/receivingGoodsImage/topTab.png);
}</string>
           </property>
           <property name="text">
            <string>未完成</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="finishedRecieveButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	font-family:&quot;微软雅黑&quot;;
	font-size:14px;
	color:#ffffff;
	border-image: url(:/receivingGoodsImage/topTab.png);
}</string>
           </property>
           <property name="text">
            <string>已完成</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="createDeliveryOrderButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	border-image: url(:/receivingGoodsImage/createDeliveryOrder.png);
}

QPushButton:pressed{
	border-image: url(:/receivingGoodsImage/createDeliveryOrder_press.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ReceiveOrdersTitleWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ReceiveOrdersListView" name="listView">
     <property name="styleSheet">
      <string notr="true">QListView{
	border:none;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>74</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>74</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#bottomWidget{
	background-color:#ffffff;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="leftWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="lastPageButton">
              <property name="minimumSize">
               <size>
                <width>87</width>
                <height>37</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>87</width>
                <height>37</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	border-image: url(:/saleImage/btn-up-nomal.png);
}

QPushButton:pressed
{
	border-image: url(:/saleImage/btn-up-pess.png);
}

QPushButton:disabled
{
border-image: url(:/saleImage/btn-up-disable.png);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>18</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1</width>
                <height>18</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 1px solid #ededed;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="nextPageButton">
              <property name="minimumSize">
               <size>
                <width>87</width>
                <height>37</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>87</width>
                <height>37</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	border-image: url(:/saleImage/btn-down-nomal.png);
}

QPushButton:pressed{
	border-image: url(:/saleImage/btn-down-pess.png);
}

QPushButton:disabled{
	border-image: url(:/saleImage/btn-down-disabled.png);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="pageLabel">
           <property name="styleSheet">
            <string notr="true">font-size:12px;
color:#999999;
font-family:&quot;微软雅黑&quot;;
padding-top:6px;</string>
           </property>
           <property name="text">
            <string>1/1页</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReceiveOrdersTitleWidget</class>
   <extends>QWidget</extends>
   <header>ReceiveOrdersTitleWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReceiveOrdersListView</class>
   <extends>QListView</extends>
   <header>ReceiveOrdersListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
